mkdocs.yml

```diff
@@ -233,7 +233,6 @@ nav:
       - 'Building question elements': 'devElements.md'
       - 'Building element extensions': 'elementExtensions.md'
       - 'Code execution': 'codeExecution.md'
-      - 'Docker packaging': 'docker.md'
```

Comment:
Do we have a replacement for this? Perhaps a `README.md` in `scripts/build_images`?